renku/core/management/datasets.py

```diff
@@ -296,7 +296,8 @@ def _add_from_url(self, dataset, dataset_path, url, link, destination):
         mode = dst.stat().st_mode & 0o777
         dst.chmod(mode & ~(stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH))
 
-        self.track_paths_in_storage(str(dst.relative_to(self.path)))
+        if self.has_external_storage:
```

Comment:
Doesn't the `ensure_external_storage` decorator already do this?

Reply:
there should probably instead be a check whether the external storage was requested...